app/src/main/java/net/squanchy/home/HomeActivity.java

```diff
@@ -51,6 +51,7 @@
 
 import static net.squanchy.google.GoogleClientId.HOME_ACTIVITY;
 
+@SuppressWarnings({"PMD.ExcessiveImports", "PMD.GodClass"})           // Unfortunately this activity has got a lot to do
```

Comment:
can we not add the suppress and increase the threshold? If we accidentally clean up this class it won't show up as a lower warning count otherwise